client/src/components/ActiveChat/Messages.js

```diff
@@ -8,20 +8,24 @@ const Messages = (props) => {
 
   return (
     <Box>
-      {messages.map((message) => {
-        const time = moment(message.createdAt).format('h:mm');
+      {messages
+        .sort(function (a, b) {
+          return a.id - b.id;
+        })
+        .map((message) => {
+          const time = moment(message.createdAt).format('h:mm');
 
-        return message.senderId === userId ? (
-          <SenderBubble key={message.id} text={message.text} time={time} />
-        ) : (
-          <OtherUserBubble
-            key={message.id}
-            text={message.text}
-            time={time}
-            otherUser={otherUser}
-          />
-        );
-      })}
+          return message.senderId === userId ? (
+            <SenderBubble key={message.id} text={message.text} time={time} />
+          ) : (
+            <OtherUserBubble
+              key={message.id}
+              text={message.text}
+              time={time}
+              otherUser={otherUser}
+            />
+          );
+        })}
```

Comment:
Configure linting/prettier to confirm the formatting styles and conventions followed by the starter project.